src/miral/minimal_window_manager.cpp

```diff
@@ -91,8 +91,12 @@ struct miral::MinimalWindowManager::Impl
     bool handle_touch_event(MirTouchEvent const* event);
 
     void apply_resize_by(Displacement movement);
+
+    static MirInputEventModifier pointer_drag_modifier;
```

Comment:
Why is this static? Why not a property in the window manager instance?

Reply:
Yeah, wasn't the best idea. Have reworked